pkg/rosa/idps.go

```diff
@@ -4,7 +4,6 @@ import (
 	"fmt"
 
 	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
-	"github.com/openshift/rosa/pkg/ocm"
```

Comment:
We can't use  `github.com/openshift/rosa/pkg/ocm` directly here, because it doesn't expose interface that we could mock. So I had to create mockable interface at our side to be able to mock the ocm calls.

The guys that are developing `github.com/openshift/rosa/pkg/ocm`  are planning to create (and I hope also expose) OCM calls as interface to increase the testability of OCM code. So we could get rid of this in the future. But I am not sure how long will it take them to do it.